testpar/t_filters_parallel.c

```diff
@@ -556,10 +556,20 @@ verify_chunk_opt_status(size_t num_dsets, test_mode_t test_mode, bool any_io, bo
              * elements (because no elements were raw data), which is what happens when performing I/O on a
              * filtered dataset with no selection. Vector I/O does report an I/O call was made if passed a raw
              * data element of size 0, so this is consistent. */
+            /* Changes: for allocation, the reported I/O is vector I/O */
```

Comment:
No need to call out "changes" in the code comment here. Just explain why we expect the different modes under each condition as  the code stands with this PR